manual/src/refman/lex.etex

```diff
@@ -31,21 +36,23 @@ let f = function
 
 \begin{syntax}
 ident: (letter || "_") { letter || "0"\ldots"9" || "_" || "'" } ;
-capitalized-ident: ("A"\ldots"Z") { letter || "0"\ldots"9" || "_" || "'" } ;
+capitalized-ident: uppercase-letter { letter || "0"\ldots"9" || "_" || "'" } ;
 lowercase-ident:
-   ("a"\ldots"z" || "_") { letter || "0"\ldots"9" || "_" || "'" } ;
-letter: "A"\ldots"Z" || "a"\ldots"z"
+   (lowercase-letter || "_") { letter || "0"\ldots"9" || "_" || "'" } ;
+letter: uppercase-letter || lowercase-letter ;
+lowercase-letter:
+"a"\ldots"z" || "š" || "ž" || "œ" || "ß" \ldots "ö" || "ø" \dots "ÿ" ;
+uppercase-letter:
+  "A"\ldots"Z" || "Š" || "Ž" || "Œ" || "Ÿ" || "À" \ldots "Ö" || "Ø" \ldots "Þ" || "ẞ"
+ ;
 \end{syntax}
 
 Identifiers are sequences of letters, digits, "_" (the underscore
 character), and "'" (the single quote), starting with a
 letter or an underscore.
 Letters contain at least the 52 lowercase and uppercase
-letters from the ASCII set. The current implementation
-also recognizes as letters some characters from the ISO
-8859-1 set (characters 192--214 and 216--222 as uppercase letters;
-characters 223--246 and 248--255 as lowercase letters). This
-feature is deprecated and should be avoided for future compatibility.
+letters from the ASCII set, and the 70 lowercase and uppercase letters
+from the ISO 8859-15 character set extended with uppercase ẞ.
```

Comment:
This is a bit confusing. 

I think you should say something like "letters contain the UTF-8 encoding of the letters of the ASCII set (U+0041-U+005A and U+0061-U+007A), letter of latin1 character sets (U+…) aswell as upper case `ẞ` (U+1E9E).

Something should also be said about normalisation or the lack thereof.

Reply:
In fact it's better to simply stick to Unicode terminology and avoid mentioning other character sets (except perhaps ASCII). So either mention only the code points or the code points and the Unicode blocks they are in, see the names on this [page](https://www.unicode.org/charts/).

Reply:
My current text is:

>Letters contain at least the 52 lowercase and uppercase letters from the ASCII
set, letters from the latin-1 complement block ("U+0080" to "U+00FF"), letters
"ŠšŽžŒœŸ" from the Latin Extended-A block ("U+0100" to "U+017F") and upper case
ẞ ("U+189E").

Reply:
Sounds good. Though the block seems to be called [`Latin-1 Supplement`](https://www.unicode.org/charts/PDF/U0080.pdf) rather than *complement*.